docs/user-guide/distributeddataclassification.rst

```diff
@@ -45,18 +61,33 @@ Let's see how ``DomainClassifier`` works in a small excerpt taken from ``example
 
     result_dataset.to_json("games_and_sports/")
 
-In the above excerpt, the domain classifier is obtained directly from `Hugging Face <https://huggingface.co/nvidia/domain-classifier>`_.
-NeMo Curator's ``QualityClassifier`` functions similarly and is obtained from `here <https://huggingface.co/nvidia/quality-classifier-deberta>`_.
+In this example, the domain classifier is obtained directly from `Hugging Face <https://huggingface.co/nvidia/domain-classifier>`_.
+It filters the input dataset to include only documents classified as "Games" or "Sports".
 
-This module functions similarly to the ``ScoreFilter`` module.
-The key differences is that it operates on the GPU instead of the CPU.
-Therefore, the Dask cluster must be started as a GPU cluster.
-Additionally, ``DomainClassifier`` requires ``DocumentDataset`` to be on the GPU with ``backend="cudf"``.
-It is easy to extend ``DistributedDataClassifier`` to your own model.
-Check out ``nemo_curator.classifiers.base.py`` for reference.
+Quality Classifier
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The Quality Classifier is designed to assess the quality of text documents, helping to filter out low-quality or noisy data from your dataset.
+
+Here's an example of how to use the ``QualityClassifier``:
+
+.. code-block:: python
+
+    from nemo_curator.classifiers import QualityClassifier
+
+    files = get_all_files_paths_under("web_documents/")
+    input_dataset = DocumentDataset.read_json(files, backend="cudf")
+
+    quality_classifier = QualityClassifier()
```

Comment:
```suggestion
    quality_classifier = QualityClassifier(filter_by=["High", "Medium"])
```

Reply:
Curious, Does this work ?

Reply:
Yes, it's been awhile since the last time I ran our quality classifier, but since it is implemented the same way as the domain classifier I would expect it to still work the same way?

Reference: https://github.com/NVIDIA/NeMo-Curator/blob/main/examples/classifiers/quality_example.py#L39

Reply:
Perfect, thanks